scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -99,6 +99,7 @@ class Data:
 def parse_idd(data):
     root = {'$schema': "http://json-schema.org/draft-04/schema#", 'properties': {}}
     data.file_size = len(data.file)
+    current_group_name = "**ungrouped**"
```

Comment:
The IDD currently starts with a group at the top, but in case that ever gets botched, this would be the group name assigned to the objects ahead of the group declaration.  More error handling could be added, but in this particular case I think it would be unnecessary to overthink this.